nifi-commons/nifi-utils/src/main/java/org/apache/nifi/processor/util/StandardValidators.java

```diff
@@ -667,6 +667,46 @@ public ValidationResult validate(final String subject, final String input, final
         };
     }
 
+    public static Validator createRegexMatchingValidatorWithEL(final Pattern pattern, final String validationMessage) {
```

Comment:
Is there a reason for creating this new method as opposed to using the existing `createRegexValidator` method with the parameter `supportAttributeExpressionLanguage` set to `true`?

Reply:
@exceptionfactory I think you mean the validator which is evaluating whether the provided regex is a valid regular expression. I'm using Regex Matching validator which is checking whether a string matches with the provided pattern. This validator is not working with parameters this is why I've added the new functionality. I might have overlooked something though, in this case can you please direct me where I can switch expression language support on for the createRegexMatchingValidator as I can only see it for createRegexValidator.

Reply:
Thanks for the clarification @timeabarna, I should have looked at the implementation more closely.  The addition looks good!

Reply:
@exceptionfactory Thanks for your help